pkg/capabilities/consensus/datafeeds/feeds_aggregator.go

```diff
@@ -0,0 +1,151 @@
+package datafeeds
+
+import (
+	"math"
+	"strconv"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/shopspring/decimal"
+	ocrcommon "github.com/smartcontractkit/libocr/commontypes"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/mercury"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type AggregatorConfig struct {
+	Feeds map[mercury.FeedID]FeedConfig
+}
+
+type FeedConfig struct {
```

Comment:
@bolekk I don't think FeedConfig/AggregatorConfig needs to be public does it? They're only used here.